pyvera/__init__.py

```diff
@@ -284,10 +284,25 @@ def __init__(self, json_obj, vera_controller):
             else:
                 self.name = 'Vera Device ' + str(self.device_id)
 
-    def get_payload_parameter_name(self, name):
+    @staticmethod
```

Comment:
I think the problem is that you can't override static methods in inherited classes.

You probably want a class method.  Not a corner of Python I've explored - but his piece looked helpful.
https://julien.danjou.info/blog/2013/guide-python-static-class-abstract-methods